core/modules/ccontrol/UserQueryFactory.cc

```diff
@@ -66,7 +66,7 @@ namespace {
 LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryFactory");
 }
 
-#define USE_ANTLR4_QUERY 0
+#define USE_ANTLR4_QUERY 1
```

Comment:
This will get reverted (back to 0) before checkin. Antlr4 will become the main parser this sprint, probably during the week of the conference, in DM-15322.